packages/desktop-client/src/components/budget/index.js

```diff
@@ -517,6 +517,7 @@ const RolloverBudgetSummary = memo(props => {
 });
 
 function BudgetWrapper(props) {
+  console.log(`<BudgetWrapper> mounted at ${new Date().toISOString()}`);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should this be removed too?